### ISSUE TITLE
stitching my own images?

### ISSUE THREAD

**Issue by winmler** [nothinglo/NISwGSP#15]
If I want to stitch my own images,what should I do?And how to generate the xxx-STITCH-GRAPH.txt?Your input-42-data contains the xxx-STITCH-GRAPH.txt!

**Comment by nothinglo** [comment]
Yes. Because I did not implement the algorithm of image matching graph right now. Also, I did not give a detailed about how to prepare the *-STITCH-GRAPH.txt yet. I will deal with these things recently. I am so sorry for your inconvenience.

If you want to stitch your own images now. You can just provide your own *-STITCH-GRAPH.txt. You can refer the format to https://github.com/nothinglo/NISwGSP/issues/9

Again, I am sorry for your inconvenience now.

**Comment by winmler** [comment]
Nono,you are fine.Thanks to your code and paper ,I got lots of harvest.  I'm looking forward to having more communication with you.
Thanks again!
